### PR TITLE
388_commit_assetGroup_test

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- Add a new sage identification integration test to do all the steps needed to send a valid ID request to Sage
- Required changess to annotation patching so that the content guid is passed and the code finds and validates the annotation for that content guid in the asset group sighting
- Fix a  few bugs and enhanced debug on the way
- Request now goes to Sage but Sage rejects it as it does not yet support the callback_detailed field on ID
---

### CODE REVIEW

File: app/extensions/auth/oauth2.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def wrapper(*args, **kwargs):
 
                 access_token = None
 
-                log.debug('Using locations = %r' % (locations,))
+                # log.debug('Using locations = %r' % (locations,))
```
Comment (hwindsor) [inline app/extensions/auth/oauth2.py:158]: These lines were coming out a lot in the debug, not adding any value so got removed

File: app/modules/asset_groups/metadata.py [hunk 2]
```diff
@@ -195,27 +199,6 @@ def validate_encounters(cls, encounters, debug):
                 )
                 owner_assignment = True
 
-            # can assign annotations (in patch only) but they must be valid
```
Comment (hwindsor) [inline app/modules/asset_groups/metadata.py:198]: This should never have been here, looks like a merge error

File: integration_tests/test_sage_detection.py [hunk 3]
```diff
@@ -1,14 +1,14 @@
 # -*- coding: utf-8 -*-
-from . import utils
```
Comment (hwindsor) [inline integration_tests/test_sage_detection.py:2]: This file should not have been changed, will fix

**Review by drewblount — APPROVED** [review]
These LGTM especially pending Sage update to accept job_id.

**Pull request merged**